src/components/Trip/TripLayout.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useMemo } from 'react'
+
+import BottomSheet from 'src/components/BottomSheet'
+import Toolbar from 'src/components/Toolbar'
+import TripContent from 'src/components/Trip/TripContent'
+import BottomSheetHeader from 'src/components/Trip/BottomSheet/BottomSheetHeader'
+import BottomSheetContent from 'src/components/Trip/BottomSheet/BottomSheetContent'
+import { getEndPlaceDisplayName } from 'src/lib/trips'
+
+const TripLayout = ({ trip }) => {
+  const toolbarNode = document.getElementById('coz-bar')
+  const title = useMemo(() => getEndPlaceDisplayName(trip), [trip])
+
+  return (
+    <>
+      <Toolbar title={title} />
+      <TripContent trip={trip} />
+      <BottomSheet
+        toolbarNode={toolbarNode}
+        header={<BottomSheetHeader trip={trip} />}
+        content={<BottomSheetContent trip={trip} />}
+      />
+    </>
+  )
+}
+
+export default React.memo(TripLayout)
```

Comment:
You should squash this commit with all the others related to the BottomSheet. ATM it's difficult to review and the commit itself is not working.

Reply:
> ATM it's difficult to review and the commit itself is not working.

On triture pas mal en faisant un peu d'explo en même temps, sur ce draft pour l'instant les commits ultra propres ne sont pas trop la priorité. Si je finis dans les temps je ferai un rebase complet pour arranger ça 👍